### PR TITLE
Added BTC Flower to Showcase.

### PR DESCRIPTION
**Pull request by evanmcfarland**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @evanmcfarland,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/383) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

**Comment by letmejustputthishere** [comment]
hey, this PR only contains two images. ideally you follow the steps outlined here, the tooling should help you create an entry for the showcase.
https://github.com/dfinity/portal#showcase-submission-guidelines

File: showcase.json [hunk 1]
```diff
@@ -2998,8 +2998,28 @@
...
-  {
+    {
+    "name": "Flower Power DAO",
+    "description": "An NFT Trilogy and community-curated garden inspired by the French Artist, Ludo.",
+    "website": "https://fpdao.app",
+    "logo": "/img/showcase/fpdao_logo.webp",
+    "screenshots": [
+      "/img/showcase/fpdao_screenshot.webp"
+    ],
+    "video": "",
```
Comment (jennifertrin) [inline showcase.json:3009]: Can you please remove the video key if you do not have a video link?

File: showcase.json [hunk 2]
```diff
@@ -2998,8 +2998,28 @@
...
+    "display": "Normal",
+    "id": "fpdao",
+    "oneLiner": "The story of blockchain, told in Art.",
+    "stats": "",
```
Comment (jennifertrin) [inline showcase.json:3013]: Can you please remove the stats key if you do not have any stats?

File: showcase.json [hunk 3]
```diff
@@ -2998,8 +2998,28 @@
...
+    "tags": [
+      "NFT",
+      "DAO"
+    ],
+    "usesInternetIdentity": true,
+    "github": "https://github.com/flowerpowerdao/",
+    "twitter": "https://twitter.com/btcflower",
```
Comment (jennifertrin) [inline showcase.json:3020]: Can you please remove the comma at the end of the Twitter key value? (It expects another key value if you have a comma at the end)

**Review by jennifertrin — CHANGES_REQUESTED** [review]

File: showcase.json [hunk 4]
```diff
@@ -2998,8 +2998,26 @@
     "youtube": "https://www.youtube.com/@omnianetwork2947",
     "submittableId": "43839848"
   },
-
-  {
+    {
+    "name": "Flower Power DAO",
+    "description": "An NFT Trilogy and community-curated garden inspired by the French Artist, Ludo.",
+    "website": "https://fpdao.app",
+    "logo": "/img/showcase/fpdao_logo.webp",
```
Comment (jennifertrin) [inline showcase.json:3005]: Hello @evanmcfarland, I believe that you uploaded PNGs. Can you please update it with the correct file type?

File: showcase.json [hunk 5]
```diff
@@ -2998,8 +2998,26 @@
...
+    "screenshots": [
```
Comment (jennifertrin) [inline showcase.json:3006]: Can you please update with the correct file type?

**Review by jennifertrin — CHANGES_REQUESTED** [review]

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**